src/main/java/Duke.java

```diff
@@ -1,10 +1,92 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+    private static void printFarewells() {
+        printHorizontalLine();
+        System.out.println("Bye. Hope to see you again soon!");
+        printHorizontalLine();
+    }
+
+    private static void printGreetings() {
+        printHorizontalLine();
+        System.out.println("Hello I'm Duke");
+        System.out.println("What can I do for you?");
+        printHorizontalLine();
+    }
+
+    private static void printHorizontalLine() {
+        System.out.println("____________________________________________________________");
+    }
+
+    private static void printLogo() {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
     }
+
+    public static void main(String[] args) {
+        printLogo();
+        printGreetings();
+        Scanner in = new Scanner(System.in);
+        String userResponse;
+        List<Task> list = new ArrayList<>();
+        while (true) {
+            userResponse = in.nextLine();
+            if (userResponse.equals("bye")) break;
+            printHorizontalLine();
+            if (userResponse.equals("list")) {
+                int size = list.size();
+                if (size == 0) {
+                    System.out.println("You have nothing on your list! Why not add one :D");
+                } else {
+                    System.out.printf("Here %s the %s in your list:%n", size > 1 ? "are" : "is", size > 1 ? "tasks" :
```

Comment:
Hello Felix! Perhaps in lines 47 and 86, you could shorten the code line a bit by setting a variable for the conditional statement, then inputting that variable into the System.out.println(). Other than that, the code looks neat!

Reply:
Hi Elizabeth! Yes, I think that might work, but I couldn't think of any appropriate variable name. Do you have some suggestions?

Reply:
Hi Felix :) Glad to see your reply! Hm, perhaps you could consider using the variable name 'taskTense'? Since the conditional statement here is to implement the appropriate tense for your task(s). Hope this helps!

Reply:
Thank you for your suggestion, Elizabeth! I will implement in the next commit :)